specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/machineLearningServices.json

```
@@ -300,6 +300,88 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{resourceName}/listresourcepreparationerrors": {
+      "post": {
+        "tags": [
+          "ProxyOperations"
+        ],
+        "operationId": "ListResourcePreparationErrors",
+        "consumes": [],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "name": "subscriptionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "resourceName",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ListResourcePreparationErrorsResult"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{resourceName}/preparenotebook": {
+      "post": {
+        "tags": [
+          "ProxyOperations"
+        ],
+        "operationId": "PrepareNotebook",
```

Similar as above, suggestion: 
```suggestion
        "operationId": "Workspaces_PrepareNotebook",
```

---

Can I change it to "Notebooks_Prepare"?